tests/data/test8

```diff
@@ -46,6 +46,7 @@ Set-Cookie: trailingspace    = removed; path=/we/want;
 Set-Cookie: nocookie=yes; path=/WE;
 Set-Cookie: blexp=yesyes; domain=%HOSTIP; domain=%HOSTIP; expiry=totally bad;
 Set-Cookie: partialip=nono; domain=.0.0.1;
+Set-Cookie: chocolate=chip; domain=curl; path=/we/want;
```

Comment:
You add a check that can filter out some cookies, and yet it ends up one more cookie used in the test. how come?

Reply:
This is adding one Set-Cookie call which should be blocked as `domain=curl;` is invalid, and it indeed isn't propagated to the cookie jar. Or am I misunderstanding you?